src/client/Microsoft.Identity.Client/Platforms/Android/AndroidPlatformProxy.cs

```diff
@@ -210,7 +210,9 @@ private static bool IsChromeEnabled()
 
         public override IBroker CreateBroker(ApplicationConfiguration appConfig, CoreUIParent uiParent)
         {
-            return new AndroidBroker(uiParent, Logger);
+            AndroidBrokerOrchestrator androidBrokerOrchestrator = new AndroidBrokerOrchestrator(uiParent, Logger);
```

Comment:
1. `orchestrator` is a type that manages other types and knows a lot of the high level flow, but doesn't know low level details. Your class isn't an orchestrator. It's a factory. I'd name it as such.
2. It's ok for factories to be static..
```AndroidBrokerFactory.CreateBroker(uiParent, logger)```

Reply:
Updated